### ISSUE TITLE
Not support TABLE-SNREF?

### ISSUE THREAD

**Issue by lz4v4** [mercedes-benz/odxtools#324]
when i change the table reference type in the example "somersault.pdx" from "`<TABLE-REF ID-REF="somersault.table.last_flip_details" />`" to "`<TABLE-SNREF SHORT-NAME="last_flip_details"/>`"

this error appears:
```
  File "/Library/Python/3.8/site-packages/odxtools/parameters/tablekeyparameter.py", line 103, in _resolve_snrefs
    self._table = resolve_snref(self.table_snref, tables, Table)
NameError: name 'Table' is not defined
```

**Comment by andlaus** [comment]
thanks for your report. can you test if #325 fixes your issue?

**Comment by lz4v4** [comment]
> thanks for your report. can you test if #325 fixes your issue?

I've tested it, and the fix in #325 works perfectly. Thanks a lot!"
